src/algebra/continued_fractions/theorems/general.lean

```diff
@@ -0,0 +1,17 @@
+/-
+Copyright (c) 2019 Kevin Kappelmann. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kevin Kappelmann
+-/
+import ..definitions.basic
+/-!
+# General Theorems for Continued Fractions
+-/
+
+namespace gcf
+variables {α : Type*}
+
+/-- Two gcfs `g` and `g'` are equal if and only if their components are equal. -/
+lemma eq_iff {g g' : gcf α} : g = g' ↔ g.h = g'.h ∧ g.s = g'.s := by { cases g, cases g', simp }
```

Comment:
Make a corresponding `@[extensionality]` lemma from the backwards direction of this iff?

Reply:
Also, for homogeneity across the library, it would be good to call this one `ext_iff`.